src/js/components/Accordion/Accordion.js

```diff
@@ -67,6 +67,6 @@ const Accordion = forwardRef(
   },
 );
 
-Accordion.propTypes = AccordionType;
+Accordion.propTypes = AccordionPropType;
```

Comment:
A naming comment ...
Since we are assigning this to "propTypes", it might be slightly more aligned to call this "AccordionPropTypes".

Reply:
Yeah that makes sense, I will go back and update